lib/components/overlays/quick_settings/widgets/qs_toggle_button.dart

```diff
@@ -36,93 +37,75 @@ class QsToggleButton extends StatefulWidget {
     this.onMenuPressed,
   });
 
-  @override
-  _QsToggleButtonState createState() => _QsToggleButtonState();
-}
-
-class _QsToggleButtonState extends State<QsToggleButton> {
   @override
   Widget build(BuildContext context) {
-    final theme = context.theme;
+    final theme = ZenitTheme.of(context);
 
-    final Color color = widget.enabled
-        ? theme.accent
-        : theme.colorScheme.background.withOpacity(0.5);
-    final String? subtitle = widget.subtitle?.resolve(enabled: widget.enabled);
+    final Color color = enabled ? theme.primaryColor : theme.surfaceColor;
+    // ignore: no_leading_underscores_for_local_identifiers
```

Comment:
remove the lint ignore and rename the variable, you can use this.subtitle for the class field so you can avoid name collision